geoapps/octree_creation/__init__.py

```diff
@@ -5,5 +5,6 @@
 #  geoapps is distributed under the terms and conditions of the MIT License
 #  (see LICENSE file at the root of this source code package).
 
-from .application import OctreeMesh
+from .constants import app_initializer
+from .driver import OctreeDriver
```

Comment:
I think that the import of driver here creates the import warning. 
![image](https://user-images.githubusercontent.com/55204635/163241342-8edb4301-02ac-42f5-9b87-2d8265599928.png)
Don't think we need it.